docs/generator/links_to_types.go

```diff
@@ -84,7 +84,7 @@ func outputComponentsSection(name string, meta metadata.Metadata) string {
 	for _, outputDataType := range meta.AllTypesExported() {
 		if list := allComponentsThatAccept(outputDataType); len(list) > 0 {
 			section += fmt.Sprintf(
-				"- Components that consume [%s]({{< relref \"../compatibility/%s\" >}})\n",
+				"- Components that consume [%s](../compatibility%s)\n",
```

Comment:
We need to go up one more directory to get to the parent of `compatibility` from these component pages.

I think we've lost the trailing slash accidentally. Although it was not desirable in the `relref` (https://grafana.com/docs/writers-toolkit/review/doc-validator/errata/#relref-has-trailing-slash), it avoids a redirect if you are using URLs instead.

```suggestion
				"- Components that consume [%s](../../compatibility/%s)\n",
```

Reply:
Ooops I missed testing that link in each component topic... I focused just on `compatibility/_index.md`  Fixed and tested.